autoptsclient-zephyr.py

```diff
@@ -53,15 +53,6 @@ def check_args(self, args):
     def init_iutctl(self, args):
         autoprojects.iutctl.init(args)
 
-    def setup_project_pixits(self, ptses):
-        autoprojects.gap.set_pixits(ptses[0])
-        autoprojects.dis.set_pixits(ptses[0])
-        autoprojects.sm.set_pixits(ptses[0])
-        autoprojects.l2cap.set_pixits(ptses[0])
-        autoprojects.gatt.set_pixits(ptses)
-        autoprojects.mesh.set_pixits(ptses)
-        autoprojects.mmdl.set_pixits(ptses)
-
     def setup_test_cases(self, ptses):
         self.test_cases = autoprojects.gap.test_cases(ptses[0])
```

Comment:
You could always pass all PTS instances to `profile.test_cases()`
Also, I think ideally you wouldn't need this setup_test_cases in here but in a common client.

Reply:
But maybe that could be done in a next PR.

Reply:
Also see if we can remove `kernel_image` argument.

Reply:
Sure, I'll see what we can do :+1: